released/index.yaml

```
@@ -23,11 +23,21 @@ entries:
   tpsvc-remote-engine:
   - apiVersion: v1
     appVersion: 1.0.0-alpha1
-    created: "2019-12-13T11:26:37.027757+01:00"
+    created: "2020-05-18T22:45:44.396984+02:00"
```

```suggestion
        created: "2019-12-13T11:26:37.027757+01:00"
```
I would suggest to keep the date if the content/version does not change